core/src/Instance/AdfsAuthority.cs

```diff
@@ -134,7 +147,7 @@ protected override void AddToValidatedAuthorities(string userPrincipalName)
             catch (Exception exc)
             {
                 const string msg = "On-Premise ADFS enrollment server endpoint lookup failed. Error - ";
-                string noPiiMsg = exc.GetPiiScrubbedDetails();
+                string noPiiMsg = CoreExceptionService.Instance.GetPiiScrubbedDetails(exc);
```

Comment:
The previous implementation was conceptually imprecise because an `Exception` instance does not necessarily have a method `GetPiiScrubbedDetails()`. This new implementation happens to eliminate that problem. 😄 👍